modules/oe_whitelabel_modal_page/README.md

```diff
@@ -0,0 +1,26 @@
+# OpenEuropa Whitelabel Modal Page
+
+This module provides integration with the [Modal Page](https://www.drupal.org/project/modal_page) contrib module, enabling modal dialogs and toast notifications styled with the Bootstrap Component Library (BCL).
+
+## Features
+
+- **Bootstrap Component Library integration**: Template overrides that render modals and toasts using BCL components
+- **Permission-based form simplification**: Hides advanced modal configuration fields from users without the "administer advanced modal page configuration" permission
+- **Library conflict prevention**: Automatically prevents modal_page from loading its own Bootstrap libraries to avoid conflicts with BCL
```

Comment:
It should be mentioned that two of these points are now implemented in the theme itself, independent of this module.

(I don't know if the top-level README.md has place for this.)